default.yml

```diff
@@ -1,6 +1,7 @@
 AllCops:
   Exclude:
-    - "db/schema.rb"
+    - db/schema.rb
+    - vendor/**/*
```

Comment:
_I use `rbenv` instead of `rvm` and [follow a pattern from Ryan McGeary](http://ryan.mcgeary.org/2011/02/09/vendor-everything-still-applies/) to `bundle install --path vendor` instead of using gemsets. In addition to the benefits Ryan lays out in that article, this also allows me to easily search through all of the project's gems if I'm not sure where a method is defined. Obviously, I don't want to know about offenses in gems._

Reply:
When developing in a Docker container, I've found that bundling to vendor is helpful as well. This allows you to add a gem to your project and then just run bundle instead of rebuilding your entire container.